src/lucky/mount_component.cr

```diff
@@ -1,19 +1,46 @@
 module Lucky::MountComponent
+  # Appends the `component` to the view.
+  #
+  # When `Lucky::HTMLPage.settings.render_component_comments` is
```

Comment:
Maybe: "When `Lucky::HTMLPage.settings.render_component_comments` is set to true" Since that is what the user will set. They won't know about the private method

Reply:
That's what it says now though... 🤔

Reply:
Just testing you. You passed 🤣LOL

Reply:
Facepalm.gif